lld/test/ELF/aarch64-reloc-pauth.s

```diff
@@ -0,0 +1,228 @@
+# REQUIRES: aarch64
+
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+# RUN: llvm-mc -filetype=obj -triple=aarch64 %p/Inputs/shared2.s -o main.so.o
```

Comment:
When we use `main` in a test, we typically indicate that it belongs to the main executable.
A .o file linked into a DSO should not use a confusing name `main.so.o`. You can use a.o or b.o.

Reply:
Fixed, thanks, see d32b8e32fc1c938e5a005997b832d215a308037d